.github/workflows/compatibility_checker.yml

```diff
@@ -48,6 +53,7 @@ jobs:
 
       # Wipe database from previous runs
       - name: Wipe SuiDB
+        if: ${{ github.event.inputs.clean == true }}
```

Comment:
Inputs.clean works as well

Reply:
Does it? I've seen some weird behavior with Boolean check truthiness in this temp laying syntax.. hence why I did the explicit form here.